packages/iris-grid/src/IrisGridTableModel.js

```diff
@@ -313,7 +313,7 @@ class IrisGridTableModel extends IrisGridModel {
   }
 
   get isTotalsAvailable() {
-    return true;
+    return this.table.getTotalsTable != null;
```

Comment:
Curious why you're loose checking null and not strict `undefined`.

Reply:
No reason in particular. It wouldn't work if it was null anyway.

One day I'll convert all the grid stuff to TypeScript, and it'll be easier to be strict checking.